### ISSUE TITLE
(TinyViT) How to get the logits of CLIP

### ISSUE THREAD

**Issue by jianwang-scu** [microsoft/Cream#110]
Hi, 
When I finished reading the paper, I had a question: how did CLIP get the logits? Because CLIP doesn't have a classification header.
I glanced at the codes roughly, it was hard for me to make sense of their logic in a short time because the amount of work was huge. What a brilliant job. Thanks for your sharing!

**Comment by wkcn** [comment]
Thanks for your attention to our work ! : )

We finetune the vision backbone (vit-large/14) of CLIP on ImageNet-22k, and obtain the model `CLIP-ViT-Large/14-22k`, which has the 22k classification head. The model can be evaluated on IN-1k directly (by mapping 22k labels to 1k labels) and achieves
85.9% top-1 accuracy. It is a teacher model of TinyViT.

You can download `CLIP-ViT-Large/14-22k` from the tutorial [Save Teacher Logits](https://github.com/microsoft/Cream/blob/main/TinyViT/docs/SAVE_TEACHER_LOGITS.md).

**Comment by jianwang-scu** [comment]
Thanks a lot!  : )
